src/order/basic.lean

```diff
@@ -770,6 +770,22 @@ instance : densely_ordered punit := ⟨λ _ _, false.elim⟩
 
 end punit
 
+section prop
+
+instance Prop.has_le : has_le Prop := ⟨(→)⟩
```

Comment:
This could maybe do with a docstring. Bonus points if it references the strongest order instance we eventually put on `Prop`.